tools/gulp/tasks/aot.ts

```diff
@@ -27,7 +27,7 @@ task('aot:pre', sequenceTask(
 );
 
 task('aot:deps', [], execTask(
-  'npm', ['install'], {cwd: demoAppSource}));
+  'yarn', ['--cwd', demoAppSource]));
```

Comment:
What's the issue with the `cwd` option?

Reply:
Nothing! This resulted from a mishap where I had accidentally deleted the package.json in the app directories, and this was one attempt to fix it. But in general, I just think this is cleaner. Do you think I should change it back, or is this more a "why do this?" comment?

Reply:
Both seems fine, it's just personally I prefer to keep the `args` array limited to the desired action.

Spawning a process with the proper CWD makes this possible and works perfectly.

Reply:
Agreed, will change back